2012-04-20	Nicolas Setton  <setton@adacore.com>

	* gtk/gtkthemingengine.c (set_stroke_style): nearbyint
	is not available on x86-solaris: use rint instead, suitable
	on Solaris.

*** gtk/gtkthemingengine.c.orig	Fri Apr 20 14:51:01 2012
--- gtk/gtkthemingengine.c	Fri Apr 20 14:51:15 2012
***************
*** 1481,1487 ****
        /* Optimize the common case of an integer-sized rectangle
         * Again, we care about focus rectangles.
         */
!       if (n == nearbyint (n))
          {
            segments[0] = 1;
            segments[1] = 2;
--- 1481,1487 ----
        /* Optimize the common case of an integer-sized rectangle
         * Again, we care about focus rectangles.
         */
!       if (n == rint (n))
          {
            segments[0] = 1;
            segments[1] = 2;
